applications/nrf_desktop/doc/dfu.rst

```diff
@@ -47,19 +47,48 @@ If the buffer is small, the host must perform the DFU progress synchronization m
 MCUboot bootloader mode
 =======================
 
-The MCUboot bootloader can either move the image to the primary slot before booting it (``swap mode``) or boot the image directly from the secondary slot (``direct-xip mode``).
+This module supports the following modes of the MCUboot bootloader and reports the corresponding bootloader name:
 
-If the MCUboot bootloader in the swap mode is selected, the DFU module does the following:
+* ``swap`` (:kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_SWAP_USING_MOVE`) - ``MCUBOOT``
+* ``direct-xip`` (:kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP`) - ``MCUBOOT+XIP``
+* ``RAM load`` (:kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_RAM_LOAD`) - ``MCUBOOT``
+
+The DFU module relies on the listed Kconfig options that indicate the MCUboot bootloader mode to determine the correct behavior.
+
+.. note::
+   Other bootloader modes might be compatible with this module, but they are not actively tested as part of the nRF Desktop configurations.
+
+See the :ref:`nrf_desktop_bootloader` section for more information on the MCUboot bootloader configuration.
+
+Swap mode
+---------
+
+MCUboot in the swap mode moves the image to the primary slot before booting it.
+
+If this bootloader mode is selected, the DFU module does the following:
 
  * Requests the image upgrade after the whole image is transferred over the :ref:`nrf_desktop_config_channel`.
  * Confirms the running image after the device is rebooted.
 
-If the MCUboot bootloader's direct-xip mode is used, the module does not mark the newly uploaded image as pending and does not confirm it after a successful boot.
+Direct-xip mode
+---------------
+
+MCUboot in the direct-xip mode boots the image directly either from the primary or from the secondary slot.
```

Comment:
We could even skip information about what MCUboot in the given mode does under the hood (already described in more general nRF Desktop bootloader section).